velox/exec/LocalPlanner.cpp

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 #include "velox/exec/LocalPlanner.h"
+#include "AssignUniqueId.h"
```

Comment:
should be full paths `velox/exec/AssignUniqueId.h`

Reply:
+1